build.sbt

```diff
@@ -177,7 +177,7 @@ lazy val app = mainProject("app")
       val alephiumHome = "/alephium-home"
 
       new Dockerfile {
-        from("eclipse-temurin:17-jre")
+        from("eclipse-temurin:21-jre")
```

Comment:
should we update our ci workflow too? we are only testing against [java 11](https://github.com/alephium/alephium/blob/6d5648ff43fa870afdf90886ec23492cb31de8de/.github/workflows/test.yml#L15)
In the [doc also](https://wiki.alephium.org/full-node/getting-started#using-java)